<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fct_comp_1</class>
 <widget class="QDialog" name="fct_comp_1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gérer les inscriptions à des épreuves</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Inscriptions</string>
          </attribute>
          <widget class="QTableWidget" name="tableWidget">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>521</width>
             <height>381</height>
            </rect>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="lineWidth">
            <number>1</number>
           </property>
           <property name="textElideMode">
            <enum>Qt::ElideMiddle</enum>
           </property>
           <property name="rowCount">
            <number>12</number>
           </property>
           <property name="columnCount">
            <number>2</number>
           </property>
           <attribute name="horizontalHeaderCascadingSectionResizes">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderMinimumSectionSize">
            <number>23</number>
           </attribute>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>249</number>
           </attribute>
           <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderCascadingSectionResizes">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderDefaultSectionSize">
            <number>33</number>
           </attribute>
           <row>
            <property name="text">
             <string>1</string>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </row>
           <row/>
           <row/>
           <row/>
           <row/>
           <row/>
           <row/>
           <row/>
           <row/>
           <row/>
           <row/>
           <row/>
           <column>
            <property name="text">
             <string>Numéro</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Pays</string>
            </property>
           </column>
           <item row="0" column="0">
            <property name="text">
             <string/>
            </property>
           </item>
          </widget>
          <widget class="QWidget" name="horizontalLayoutWidget">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>420</y>
             <width>521</width>
             <height>41</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QPushButton" name="pushButton_2">
              <property name="text">
               <string>Ajouter un participant</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_3">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Supprimer les inscriptions sélectionnés</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QLabel" name="label">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>320</x>
             <y>400</y>
             <width>191</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Nombre d'inscriptions: &lt;/span&gt;0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
          </widget>
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>50</x>
             <y>400</y>
             <width>160</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Épreuve:&lt;/span&gt; ---&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Numéro de l'épreuve:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Sélectionner</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>fct_comp_1</receiver>
   <slot>refreshResult()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableWidget</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>fct_comp_1</receiver>
   <slot>refreshSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>287</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>fct_comp_1</receiver>
   <slot>removeInscription()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>414</x>
     <y>526</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>fct_comp_1</receiver>
   <slot>OpenAddParticipant()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>526</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>refreshResult()</slot>
 </slots>
</ui>
